dist/main.js

```diff
@@ -0,0 +1,38 @@
+const dataManager = new DataManager();
+const renderer = new Renderer();
+
+const imgError = (image) => {
+  image.src = '../assets/img/alt-player.png';
+};
+
+$('#load-btn').on('click', function () {
+  let input = $('#input').val();
+  dataManager.loadTeamPlayers(input);
+});
+
+$('#display-btn').on('click', function () {
+  renderer.renderTeam(dataManager.data.currentTeam);
+});
```

Comment:
you should have one button to load AND display the players... same for the dream team.

you should have returned the data from the server call (in the DataManager) and then use the render function with that data inside a callback function